generator/analysis_options.yaml

```diff
@@ -13,15 +13,18 @@
 
 include: package:lints/recommended.yaml
 
-# Uncomment the following section to specify additional rules.
+analyzer:
+  language:
+    strict-raw-types: true
 
-# linter:
-#   rules:
-#     - camel_case_types
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
 
-# analyzer:
-#   exclude:
-#     - path/to/excluded/files/**
+linter:
+  rules:
+    avoid_relative_lib_imports: false
+    package_api_docs: true
```

Comment:
Should we enable this for the generator?

Reply:
Fixed